doc/migrations/3.3.0_3.3.0plus.md

```diff
@@ -51,7 +51,44 @@ TODO
 
 ### Riak
 
-TODO
+Changes to Riak schema are backwards compatible with the current MongooseIM release.
```

Comment:
Changes to Riak schema are backward compatible with the current MongooseIM release.